eng/scripts/build.ps1

```diff
@@ -16,19 +24,33 @@ if (![string]::IsNullOrWhiteSpace($filter)) {
     $keys = $keys.Where( { $_ -match $filter }) 
 }
 
-$keys | ForEach-Object { $sdks[$_] } | ForEach-Object -Parallel {
+$keys | ForEach-Object { $sdks[$_] } | ForEach-Object {
```

Comment:
Similar to my above comment, the script is small right now so I don't think it matters, but I would consider moving all the script logic into various functions before it gets too large. At least on the engsys side we have some really hard to maintain scripts right now that have grown very large without any refactoring.